### ISSUE TITLE
"Tool" template doesn't have any entitlements

### ISSUE THREAD

**Issue by Muirey03** [theos/theos#413]
On iOS 11+, "tools" are required to have these entitlements otherwise they will crash with `Killed: 9`:
```
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
    <key>platform-application</key>
    <true/>
    <key>com.apple.private.security.container-required</key>
    <false/>
</dict>
</plist>
```
Please could you put these entitlements into an ent.xml, and add them to the tool template (setting `CODESIGN_FLAGS` of course) so that new tools don't immediately crash when you try to run them?

**Comment by kabiroberai** [comment]
I agree with this change (at least the platform-application part; setting container-required to false may be a breaking change in some cases). @kirb @uroboro thoughts?

**Comment by uroboro** [comment]
Thinking about backwards compatibility, having extra entitlements shouldn’t be an issue, right?

**Comment by kabiroberai** [comment]
Yeah but the no-container entitlement will change whether tools are containerised by default (although I’m not sure whether it’s possible for tools to have containers in the first place, so maybe the entitlement is needed)

**Comment by Muirey03** [comment]
I don't believe binaries in /bin or /usr/bin have containers, correct. The entitlement is actually mainly for rootlessJB3, as executables in /var/containers do require this entitlement (as explained here: https://github.com/jakeajames/rootlessJB3). Adding the entitlement will not have any effect on binaries in /usr/bin or /bin, but is required for binaries in /var/containers so I don't see any reason not to include it.
